manifests/install.pp

```diff
@@ -25,10 +25,10 @@
   }
 
   if $python::manage_python_package {
-    package { 'python':
+    ensure_packages (['python'],
```

Comment:
I don't think I like this. There's already a condition around the package resource, so it shouldn't be required to wrap it in `ensure_packages()`

Reply:
This is part of @acullenn 's code changes and i am unsure why this was added.
If they do not reply i will see if i can revert this.